pages/[lng]/cart/index.tsx

```diff
@@ -1,7 +1,7 @@
 /* library package */
 import { FC, useState } from 'react'
 import { GetServerSideProps, InferGetServerSidePropsType } from 'next'
-import { useI18n, useCart } from '@sirclo/nexus'
+import { useI18n, useCart, useAuthToken } from '@sirclo/nexus'
```

Comment:
jadi new lines kalo udh lebih dr 2 import, yg files lain juga yaa
```suggestion
import {
 useI18n,
 useCart, 
 useAuthToken 
} from '@sirclo/nexus'
```